lighthouse-core/lib/lh-error.js

```diff
@@ -57,23 +58,14 @@ class LighthouseError extends Error {
     super(errorDefinition.code);
     this.name = 'LHError';
     this.code = errorDefinition.code;
-    this.friendlyMessage = str_(errorDefinition.message, properties);
+    const mergedProperties = {...errorDefinition.defaultProperties, ...properties};
+    this.friendlyMessage = str_(errorDefinition.message, mergedProperties);
     this.lhrRuntimeError = !!errorDefinition.lhrRuntimeError;
     if (properties) Object.assign(this, properties);
 
     Error.captureStackTrace(this, LighthouseError);
   }
 
-  /**
-   * @param {LighthouseError} err
-   * @return {LighthouseError}
-   */
-  static fromLighthouseError(err) {
```

Comment:
was this just never used before? I'm not sure why it would be needed so seems good haha :)

Reply:
It was, but then the only caller was deleted and this was kept!